### DISCUSSION TITLE
Are Environment Variables recorded in Vercel accessible during the run step?

### DISCUSSION THREAD

**Discussion by jacobtkoch** [vercel/community#769] [Help] [answered]
Are env variables stored in the Vercel only available during the Build step? Or are they part of the container and can be accessed in the run step? For example, if 'yarn start' had some custom commands that needed an env var, could this env var be set within Vercel and accessible via the 'yarn start'? This question hinges on 'yarn start' being called as the run command, per my [other question](https://github.com/vercel/community/discussions/768).

**Comment by leerob** [comment, marked as answer]
The Environment Variables on Vercel are accessible during the build step, as well as during runtime (for example, inside of a Vercel Function). I commented back on your other question as well 😄
